### DISCUSSION TITLE
Tags for First Time Contributors

### DISCUSSION THREAD

**Discussion by vladimirfomene** [interledger/rafiki#65] [Q&A]
Hello @matdehaast , @sharafian , @cairin , @sentientwaffle, @wilsonianb ,

Can we create tags for first time contributors like me, it will help us quickly get started. Maybe we can also use tags to categorize what the internal team at Coil will work on and what is open for contribution. I also having some issues setting up my dev environment on my laptop. What will be the best place to reach out for help?

Thanks in advance,
Vladi

**Comment by cairin** [comment]
> Can we create tags for first time contributors like me

I have cleaned up the labels, and added some documentation in the [contribution doc](https://github.com/coilhq/rafiki/blob/main/.github/contributing.md#labels). You will also now see good first issues on the [contribute page](https://github.com/coilhq/rafiki/contribute).

> I also having some issues setting up my dev environment on my laptop. What will be the best place to reach out for help?

This is the best place to ask for help, because it allows other people who run into the same issue to see the solution :tada: 

Can you elaborate on what issues you're having?

**Reply by vladimirfomene** [reply]
Thanks @cairin. I have seen the changes you made to the repo. I look forward to the posting of "good first issues". In the meantime, I forked the repository, clone it and tried running the following command: `yarn install --immutable --immutable-cache` and I got the following error. 
<img width="952" alt="error-rafiki" src="https://user-images.githubusercontent.com/11140070/127631346-d3e257d3-c9db-438f-979c-15f7c5e850ca.PNG">

**Reply by cairin** [reply]
You shouldn't need to run `yarn install --immutable --immutable-cache` on your local machine. Just `yarn install` should be fine - the CI will run this step to verify that the dependencies are all in order. We do use zero installs so you can probably skip this step.

After that you should be able to run `yarn lint` or `yarn test`. And if you want to run tests in a specific package you can use `yarn workspace <packagename> test` of just `cd` into the package and call `yarn test` from there to just test that package.

**Reply by vladimirfomene** [reply]
Thanks @cairin. I have been able to install all the dependencies as advised. I just realized that we don't yet have first time issue in the issues tab. Is there something you are working on that I could support you with? Even if it is documentation, I'm open to it.
